annualreports/2025.md

```diff
@@ -75,6 +75,8 @@ The FPA has awarded in 2024 a total of USD 82 350.00 to developers, some through
 | Obelisk              | Modernize, revise FreeCAD Art Guidelines and Icons                 | UX and design | USD 1750 |
 | kadet1090            | Unified Transparent Previews                                       | UX and design | EUR 1500 |
 
+Unfortunately, we have farewell to the bgbsww, which we have been in contact with many times during the year. In order to keep the name alive, we decided to give the name of the 1.0 version that has been developed for years with the community. We commemorate him with gratitude and love from here. And here we report that we are not only financially but also in a familial relationship with the people who are given grants.
```

Comment:
It is with great regret that we say farewell to bgbsww, who died in late 2024, and whose contributions were invaluable to the Toponaming project (among others). He received several grants from the FPA, and we worked closely with him throughout the year. FreeCAD 1.0 is dedicated to his memory, and his contributions to the project will live on in the work of everyone who enjoys the model stability that his work was instrumental in providing.